tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
```

Comment:
Why is it inside this project?

Reply:
tsconfig describes a project for typescript, so every project must have it. However, since we want most rules to be consistent, it would be nice to have a base tsconfig and extend it.

https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#configuration-inheritance-with-extends
https://github.com/Microsoft/TypeScript/issues/18865